samples/unit-tests/chart/zoomtype/demo.css

```diff
@@ -3,3 +3,7 @@
     height: 400px;
     margin: 0 auto;
 }
+
+.flex-container {
+    display: flex;
+}
```

Comment:
This will not work in Karma - CSS file (all together with HTML file) is not loaded